tests/apollo/util/bft.py

```diff
@@ -336,6 +341,10 @@ async def change_configuration(self, config):
             # Generate certificates for replicas, clients, and reserved clients
             self.generate_tls_certs(self.num_total_replicas() + config.num_clients + RESERVED_CLIENTS_QUOTA)
 
+        # remove existing transaction signing keys and generate again
```

Comment:
this comment can be removed I think

Reply:
Is it okay if I keep this one? The `change_configuration` method has a few comments, and this probably adds some value to what is being done. Let me know.

Reply:
ok. will leave it.